README.md

```diff
@@ -1,49 +1,49 @@
 # GenPipes in a container
 
-You can use GenPipes in a Container (GiaC) to run GenPipes on a single machine, on a a Torque/PBS cluster or on a SLURM cluster.
+You can use GenPipes in a Container (GiaC) to run GenPipes on a single machine, on a Torque/PBS cluster or on a SLURM cluster.
 
-If [Singularity](https://github.com/hpcng/singularity) is installed on your LINUX machine you are all set, a simple user with no special privilege is enough (no sudo needed).
-
-While you can use (GiaC) to debug and develop GenPipes on your laptop, [GenPipes](https://bitbucket.org/mugqic/genpipes/src/master/README.md) is design to run analysis on Super Computers.  
+If Singularity/Docker is installed on your LINUX machine you are all set, a simple user with no special privilege is enough (no sudo needed).
 
+While you can use (GiaC) to debug GenPipes on your laptop, [GenPipes](https://bitbucket.org/mugqic/genpipes/src/master/README.md) is design to run analysis on Super Computers.
 
 ## Install a compatible container technology on your machine
-Follow installation procedure from the  [Singularity install page](ttps://github.com/hpcng/singularity/blob/master/INSTALL.md)
 
+Follow installation procedure from the [Singularity install page](https://github.com/hpcng/singularity/blob/master/INSTALL.md) or the [Docker install page](https://docs.docker.com/get-docker/).
```

Comment:
We should point to apptainer directly and let singularity stay there for compatibility but talk about apptainer in the doc.

https://apptainer.org/docs/user/main/quick_start.html
or https://github.com/apptainer/apptainer/blob/main/INSTALL.md
for install link